tests/unit/RestorationType.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { shallowWrapperFactory } from '../jest-wrapper-factory'
+import RestorationType from '@/components/Restoration/RestorationType.vue'
+import { RelationshipsPanel } from '@bcrs-shared-components/relationships-panel'
+import { LimitedRestorationPanel } from '@bcrs-shared-components/limited-restoration-panel'
+
+describe(`Initialize Restoration Type Component`, () => {
+  it('renders the component properly', () => {
+    const wrapper = shallowWrapperFactory(
+      RestorationType
+    )
+
+    // verify the component is rendered
+    expect(wrapper.find('#restoration-type').exists()).toBe(true)
+
+    // verify sub-components on page are rendered
+    // expect(wrapper.findComponent(RelationshipsPanel).exists()).toBe(true)
+    // expect(wrapper.findComponent(LimitedRestorationPanel).exists()).toBe(true)
+  })
+})
```

Comment:
If this sub-component has appropriate tests in the shared repo, then you don't need to test it again here.

However, the component that hosts this sub-component should make sure the sub-component is displayed when needed, and also verify that the right thing is done with any events (or getters/computed variables).

Reply:
Yes, I was confused what should I exactly test since I've done a lot of the testing in the sub-components (bcrs shared repo).

Reply:
Then you don't need to test the sub-component again.

But you should test the props you would send to it and how you handle the events you would get back from it -- we don't do this nearly as well as we should, but we should try :)

Think "tiers" -- THIS component should be tested here, but not its parent or children components.

Reply:
If extra unit tests will take more than 1 day, I'm OK with creating a new Tech Debt ticket to expand them later.